doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -134,6 +134,11 @@ Bluetooth samples
 
   * Added a workaround for nRF5340 revision 1 Errata 117.
 
+* :ref:`peripheral_fast_pair` sample:
+
+  * Added possibility of toggling the advertising modes.
```

Comment:
Maybe: `Added possibility of toggling between show and hide UI indication in Fast Pair not discoverable advertising.`

Reply:
Toggling between discoverable and not discoverable advertising was introduced before.